pkg/cluster/check/default.go

```diff
@@ -13,57 +13,124 @@ import (
 	"github.com/siderolabs/talos/pkg/machinery/config/machine"
 )
 
+// checks is a map containing all our cluster checks defined once.
+var checks = map[string]ClusterCheck{
```

Comment:
let's not use global maps here, it can be a private function with a big `switch` statement and panic on `default:` case

Reply:
and check names should be constants